maro/cli/inspector/cim_dashboard.py

```diff
@@ -0,0 +1,473 @@
+import math
+import os
+
+import altair as alt
+import numpy as np
+import pandas as pd
+import streamlit as st
+
+import maro.cli.inspector.dashboard_helper as helper
+
+from .params import CIMItemOption
+from .params import GlobalFilePaths as Gfiles
+from .params import GlobalScenarios
+from .visualization_choice import CIMIntraViewChoice, PanelViewChoice
+
+
+def start_cim_dashboard(source_path: str, epoch_num: int, prefix: str):
+    """Entrance of cim dashboard.
+
+    Args:
+        source_path (str): Data folder path.
+        epoch_num (int) : Number of data folders.
```

Comment:
total number of epoches, i.e. the total number of data folders since there is a folder per epoch?

Refine the wording.